File: Number of 1 Bits/java/code.java
class Solution {
    public int hammingWeight(int n) {
        int size = 32;
        int counter = 0;

        int i = 0;
        while (i < size) {
            if ((n & 1) == 1) {
                counter++;
            }
            n = n >> 1;
            i++;
        }
        return counter;
    }
}

File: Number of 1 Bits/php/code.php
class Solution {

/**
 * @param Integer $n
 * @return Integer
 */
function hammingWeight($n) {
    $counter = 0;
    $size = 32;
    
    for ($i = 0; $i < $size; $i++) {
        if (($n & 1) == 1) {
            $counter++;
        }
        $n = $n >> 1;
    }
    
    return $counter;
}
}
